userdocs/src/usage/minimum-iam-policies.md

```diff
@@ -0,0 +1,145 @@
+# Minimum IAM policies
+
+This document describes the minimum IAM policies needed to run the main use cases of eksctl. These are the ones used to
+run the integration tests.
+
+> **Note**: remember to replace `<account_id>` with your own.
+
+AmazonEC2FullAccess
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": "ec2:*",
+            "Effect": "Allow",
+            "Resource": "*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": "elasticloadbalancing:*",
+            "Resource": "*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": "cloudwatch:*",
+            "Resource": "*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": "autoscaling:*",
+            "Resource": "*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": "iam:CreateServiceLinkedRole",
+            "Resource": "*",
+            "Condition": {
+                "StringEquals": {
+                    "iam:AWSServiceName": [
+                        "autoscaling.amazonaws.com",
+                        "ec2scheduled.amazonaws.com",
+                        "elasticloadbalancing.amazonaws.com",
```

Comment:
HI @martina-if, I know this is already merged, but we found that this also needs to be included (using eksctl v0.21.0): `"eks.amazonaws.com",`

Reply:
Thanks @kr3cj, I updated it [here](https://github.com/weaveworks/eksctl/pull/2267#discussion_r437014109)

If you find more things that need updating feel free to open a PR if you like!

Reply:
Thanks @martina-if , will do.